<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColumnTest</class>
 <widget class="QWidget" name="ColumnTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1319</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColumnTest</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_19" stretch="6,1">
     <item>
      <widget class="QWidget" name="widget_5" native="true">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="4">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>5SB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="MyColumn" name="myColumn_5"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num5">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>1SB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="MyColumn" name="myColumn_3"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num3">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string>1ZB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="MyColumn" name="myColumn_9"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num9">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>11SB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="MyColumn" name="myColumn_8"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num8">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>3AB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="MyColumn" name="myColumn_2"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num2">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>9SB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="MyColumn" name="myColumn_7"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_7">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="8">
         <widget class="QGroupBox" name="groupBox_10">
          <property name="title">
           <string>2ZB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="MyColumn" name="myColumn_18"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num18">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="QGroupBox" name="groupBox_14">
          <property name="title">
           <string>12SB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <widget class="MyColumn" name="myColumn_17"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num17">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_16">
          <property name="title">
           <string>4AB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <item>
            <widget class="MyColumn" name="myColumn_11"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num11">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>7SB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="MyColumn" name="myColumn_6"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num6">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QGroupBox" name="groupBox_12">
          <property name="title">
           <string>6SB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="MyColumn" name="myColumn_14"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num14">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_15">
          <property name="title">
           <string>2AB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <widget class="MyColumn" name="myColumn_10"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num10">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QGroupBox" name="groupBox_18">
          <property name="title">
           <string>4SB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_18">
           <item>
            <widget class="MyColumn" name="myColumn_13"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num13">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QGroupBox" name="groupBox_11">
          <property name="title">
           <string>8SB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="MyColumn" name="myColumn_15"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num15">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QGroupBox" name="groupBox_13">
          <property name="title">
           <string>2SB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <widget class="MyColumn" name="myColumn_12"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num12">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>3SB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="MyColumn" name="myColumn_4"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num4">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>1AB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="MyColumn" name="myColumn_1"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num1">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QGroupBox" name="groupBox_17">
          <property name="title">
           <string>10SB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <item>
            <widget class="MyColumn" name="myColumn_16"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edt_num16">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,5">
        <item>
         <widget class="QGroupBox" name="groupBox_20">
          <property name="title">
           <string>接收窗口</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTextBrowser" name="textBrowser"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_19">
          <property name="title">
           <string>网络设置</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1,2">
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>服务端IP地址：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="edt_ipaddress">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>端口号：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="edt_port">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="btn_connect">
                <property name="text">
                 <string>开始连接</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_disconnect">
                <property name="text">
                 <string>断开连接</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_clear">
                <property name="text">
                 <string>清除窗口</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyColumn</class>
   <extends>QWidget</extends>
   <header>mycolumn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
